HTML/html_completions.py

```diff
@@ -396,7 +404,6 @@ def get_tag_attributes():
 
     return tag_attr_dict
 
-
```

Comment:
Would you mind adding back this line to satisfy flake8?

Thanks

Reply:
No problem. Is there also a convetion for the `boolean_attributes` declaration on top of the file?

Reply:
... I feel like I should also add a extra break after the `boolean_attributes` declaration?

Reply:
Done.